### DISCUSSION TITLE
Class ctexbook Error: CTeX fontset `' could not be found. { \ctex_load_fontset: }

### DISCUSSION THREAD

**Discussion by MmdArchimedes** [nuaatug/nuaathesis#101] [Q&A]
下载了最新的版本使用demo_zh中的bachelor.tex 编译时出现如题错误，一直找不到解决的办法

**Comment by yzwduck** [comment]
试试下面两个:
1. 在 global.tex 底下添加 `\defaultfontfeatures{}`,
2. 在 bachelor.tex 顶部的文档类参数 `\documentclass[...]` 里, 添加 `fontset=windows,` (或者 ctex 能支持的 fontset 选项).
